ui/uihandler.go

```diff
@@ -101,10 +104,11 @@ func (w *HTMLEscapeWriter) Write(p []byte) (int, error) {
 // Handler is the main UI handler creating the web forms and processing them.
 func Handler(w http.ResponseWriter, r *http.Request) {
 	LogRequest(r, "UI")
-	DoExit := false
-	if r.FormValue("exit") == "Exit" {
-		log.Critf("Exit request from %v", r.RemoteAddr)
-		DoExit = true
+	DoStop := false
```

Comment:
Seems like you can't actually get a form with both 'stop' and 'load' set together? In this case maybe DoStop and DoLoad can be combined just to hold request type.

Reply:
changed to enum, thanks!